build/Makefile

```diff
@@ -101,7 +101,6 @@ $(PYTHON_DIR) :
 	. $(abspath $(DSTDIR)/bin/activate) && pip install --upgrade twisted
 	. $(abspath $(DSTDIR)/bin/activate) && pip install --upgrade pyyaml
 	. $(abspath $(DSTDIR)/bin/activate) && pip install --upgrade google
-	. $(abspath $(DSTDIR)/bin/activate) && pip install --upgrade protobuf==3.19.0
```

Comment:
aren't there a bunch of others we can remove in here?

Reply:
Yes, google. Removed!